<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GameDB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true">QListWidget::item {
	alternate-background-color: #EEEEEE;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="finishedTab">
       <attribute name="title">
        <string>Finished</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QPushButton" name="finishedNewButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>New...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="finishedListWidget">
          <property name="styleSheet">
           <string notr="true">QListWidget::item {
	alternate-background-color: #EEEEEE;
}</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="finishedTotalLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ongoingTab">
       <attribute name="title">
        <string>Ongoing</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QPushButton" name="ongoingNewButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>New...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="ongoingListWidget">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="ongoingTotalLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="droppedTab">
       <attribute name="title">
        <string>Dropped</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="3" column="0">
         <widget class="QPushButton" name="droppedNewButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>New...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QListWidget" name="droppedListWidget">
          <property name="styleSheet">
           <string notr="true">QListWidget::item {
	alternate-background-color: #EEEEEE;
}</string>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="droppedTotalLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>finishedListWidget</tabstop>
  <tabstop>finishedNewButton</tabstop>
  <tabstop>ongoingListWidget</tabstop>
  <tabstop>ongoingNewButton</tabstop>
  <tabstop>droppedListWidget</tabstop>
  <tabstop>droppedNewButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
